src/components v2/Atoms/Components/VSelect/NoOptions/NoOptions.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <label>{{ noOptionsText }}</label>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+export default Vue.extend({
+  name: 'NoOptions',
+
+  props: {
+    noOptionsText: {
+      type: String as PropType<string>,
+      default: '',
+    },
+  },

```

Comment:
**issue:** export the props so they are usuable.

You can also rename this to `text`